extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.css

```diff
@@ -85,10 +85,42 @@
   background-color: rgba(0,0,0, 1) !important;
 }
 
+.-amp-lightbox-viewer-mask {
+  z-index: 2147483643 !important;
+}
+
+.-amp-lightbox-viewer-gallery {
+  display: none;
+  z-index: 2147483645 !important;
+  top: 50px !important;
+  overflow: auto !important;
+}
+
+.-amp-lightbox-viewer[gallery-view] .-amp-lightbox-viewer-gallery {
+  display: block !important;
+}
+
+.-amp-lightbox-viewer-gallery-thumbnail {
+  display: inline-block;
```

Comment:
inline-block might have some issues with regards to whitespaces, look at how slidescroll containers and slides are done using flexboxes this might give you some hints.